test/e2e/test-launchable-smoke.sh

```
@@ -0,0 +1,591 @@
+# ══════════════════════════════════════════════════════════════════
+# Phase 2: Run brev-launchable-ci-cpu.sh
+# ══════════════════════════════════════════════════════════════════
+section "Phase 2: Run brev-launchable-ci-cpu.sh (launchable install path)"
+
+info "Running the launchable bootstrap script..."
+info "This installs Docker, Node.js 22, OpenShell, clones NemoClaw, builds CLI+plugin."
+info "Expected duration: 3-8 minutes."
+
+# The launchable script expects to run as root (it uses sudo internally).
+# On GitHub runners, we already have passwordless sudo.
+sudo -E bash "$REPO/scripts/brev-launchable-ci-cpu.sh" >"$INSTALL_LOG" 2>&1 &
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)